frontend/isoko/src/pages/SearchResults.tsx

```diff
@@ -0,0 +1,404 @@
+         ]);
+      }
+   };
+
+   // this function will requery with new tags
+   // already added to props.minorityTags
+   const applyNewTags = (e, newTags) => {
+      console.log(newTags);
+      newTags.forEach((t, i) => {
+         if (!props.minorityTags.includes(t)) {
+            props.minorityTags.push(t);
+         }
+      });
+      setShowModal(false);
+   };
+
+   // modal functions
+   const handleCloseModal = () => {
+      setShowModal(false);
+   };
+
+   return (
+      <SearchContainer>
+         <Row>
+            <Col className="col-sm-3">
+               <Sidebar>
+                  <StyledH2>Sort</StyledH2>
+                  <SidebarContainer>
+                     <SortResultsDropdown sortFunction={sortBusinesses} />
+                  </SidebarContainer>
+                  <StyledH2>Filters</StyledH2>
+                  <StyledH3>Tags</StyledH3>
+                  <TagContainer>
+                     <Row>
+                        {props.minorityTags.map((tag, index) => (
+                           <SelectedTagRow
+                              key={index}
+                              onClick={(e) =>
+                                 filterBusinesses(e, tag, filteredBusinesses)
+                              }
+                           >
+                              <MinorityTag
+                                 key={index}
+                                 name={tag}
+                                 mutable={true}
+                              />
+                           </SelectedTagRow>
+                        ))}
+                     </Row>
+                     <Row>
+                        <AddTagCol className="col-auto">
+                           <div
+                              onClick={() => {
+                                 setShowModal(true);
+                              }}
+                           >
+                              <AddTag />
+                           </div>
+                           <AddTagModal
+                              show={showModal}
+                              selectedTags={props.minorityTags}
+                              allTags={allTags}
+                              handleClose={handleCloseModal}
+                              applyNewTags={applyNewTags}
+                           />
+                        </AddTagCol>
+                     </Row>
+                  </TagContainer>
+                  <StyledH3>Open</StyledH3>
+                  <Form>
+                     <LeftDiv>
+                        <Form.Check
+                           type="checkbox"
+                           label={`Open now: ${displayTime()}`}
+                           onChange={(e) => {
+                              filterBusinesses(e, 'time', filteredBusinesses);
+                           }}
+                        />
+                     </LeftDiv>
+                  </Form>
+               </Sidebar>
```

Comment:
not quite sure what the value of that would be considering it contains most of the functionality for the SearchResults functional component, so it would end up just being another file about the same size as this one. chose to leave it in latest commit but can change if you or anyone else thinks it'd be better as another component. if that was the case i'd prob get rid of the SearchResults component and put the Sidebar component and Results Container directly in the Search page